### DISCUSSION TITLE
PROBLEM WITH FULL ISO !!

### DISCUSSION THREAD

**Discussion by ghost** [BlackArch/blackarch#4318] [Support]
Type: Discussion  + sollusion + experience

## Discussion

I've found a solution for full ISO 

WHEN ALL PACKAGES HAVE BEEN UPGRADED, PLEASE UPDATE GRUB AND DON'T FORGET WHEN THE PROCESS OF UPGRADING FIREFOX IS ALWAYS PRESENT OR CHROMIUM ,SO THAT ANY ISSUES CAN BE HANDLED, BECAUSE WHEN IT HAS BEEN UPGRADED ALL THE APPS DO NOT WORK. SO TRY TO IGNORE THE PACKAGE WHEN YOU WANT TO UPGRADE USING 
sudo nano /etc/pacman.conf 
LOOK THERE TO IGNORE PACKAGES, DON'T DELETE THEM WHEN THERE IS A CONFLICT, BECAUSE WHEN DELETING PACKAGES OR TOOLS THAT REQUIRE IT CAN CAUSE PROBLEMS FROM THE NETWORK ASPECT, TOOLS ERRORS AND MANY PROBLEMS, THE POINT IS THAT IT DOES NOT REFLECT A PERFECT ISO!! SO IT'S A SHAME TO MISS THIS OPPORTUNITY!! THANK YOU BLACKARCH YOU ARE ALWAYS THERE FOR US

**Comment by Brietutickly** [comment]
There's no reason to use the full ISO since you likely won't need all its tools; it would be better to use Arch instead.

**Comment by noraj** [comment]
https://blackarch.org/downloads.html

Reminder of the website warning:

> WARNING Installing the full ISO is highly discouraged, you will probably encounter many errors and conflicts while trying to update or even installing it. This ISO primarily targets offline installation, proceed with it only if you have a good reason, know what you are doing and are willing to spend time troubleshooting. For most people, we recommend installing either the netinstall ISO to install only what you need or the slim ISO to start with a minimal set of common tools.
